monitoring/dashboards/zookeeper.json

```diff
@@ -0,0 +1,9030 @@
+{
+  "__inputs": [
+    {
+      "name": "DS_PROMETHEUS",
+      "label": "Prometheus",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus"
+    },
+    {
+      "name": "namespace",
+      "type": "constant",
+      "label": "namespace",
+      "description": "Namespace associated with the Zenko instance",
+      "value": "zenko"
+    },
+    {
+      "name": "job",
+      "type": "constant",
+      "label": "job",
+      "description": "Name of the zookeeper job, used to filter only the zookeeper instances.",
+      "value": "artesca-data-base-quorum-headless"
+    }
+  ],
+  "templating": {
+    "list": [
+      {
+        "allValue": null,
+        "current": {},
+        "datasource": "${DS_PROMETHEUS}",
+        "definition": "label_values({job=~\"${job}\",namespace=~\"${namespace}\"}, pod)",
+        "hide": 0,
+        "includeAll": true,
+        "label": "Instance",
+        "multi": true,
+        "name": "pod",
+        "options": [],
+        "query": "label_values({job=~\"${job}\",namespace=~\"${namespace}\"}, pod)",
```

Comment:
se should not need to use regexp to filter namespace or job. we already know the exact name and don't use regexp for matching, we could simply use `job=\"${job}\"`

Reply:
not for instances (e..g `pod`) though, otherwise viewing "All" instances does not work.